lighthouse-core/gather/gatherers/seo/crawlable-anchors.js

```diff
@@ -0,0 +1,34 @@
+/**
+ * @license Copyright 2017 Google Inc. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ */
+'use strict';
+
+const Gatherer = require('../gatherer');
+const DOMHelpers = require('../../../lib/dom-helpers.js');
+
+class CrawlableAnchors extends Gatherer {
+
+  /**
+   * @param {{driver: !Object}} options Run options
+   * @return {!Promise<!Array<{href: string, text: string}>>}
+   */
+  afterPass(options) {
+    const expression = `(function() {
+      ${DOMHelpers.getElementsInDocumentFnString}; // define function on page
+      const selector = 'a[href]:not([rel~="nofollow"])';
```

Comment:
would it be better to combine this with `anchors-with-no-rel-noopener` and sort client-side for nofollow vs noopener into a single anchor gatherer? Would need to collect `rel` so that audits can filter on it

Reply:
could be `all-anchors` or something (we could do it in a followup PR if you want to keep this one focused)

Reply:
`all-anchors` SGTM, but since this will require changing `anchors-with-no-rel-noopener` I think we should do that in a followup PR as you suggested

Reply:
sg. filed as https://github.com/GoogleChrome/lighthouse/issues/3581